tests/gnrc_ipv6_nib_6ln/main.c

```diff
@@ -630,7 +630,7 @@ static void test_handle_pkt__nbr_adv__aro_not_my_eui64(void)
                                      SIXLOWPAN_ND_STATUS_SUCCESS);
     int idx;
 
-    idx = gnrc_netif_ipv6_addr_add(_mock_netif, &_loc_gb, _LOC_GB_PFX_LEN,
+    idx = gnrc_netif_ipv6_addr_add_internal(_mock_netif, &_loc_gb, _LOC_GB_PFX_LEN,
                                    GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_TENTATIVE);
```

Comment:
indentation is off here. Looks like this is the case with most of the function calls spanning multiple lines in the tests.

Reply:
Actually, this was intentinonally in this file, because otherwise I would break the line-length limit. But maybe we can find another solution.

Reply:
I was afraid of that… :(

AFAIK the aim of the indentation here is to clarify that the argument belongs to the above function. In these cases I think it the structure is clear enough. I rather have these indentation be a bit off than an increase in line length or a shortened the define (I wouldn't know where to start with that).